geowebcache/wmts/src/main/java/org/geowebcache/service/wmts/WMTSService.java

```diff
@@ -181,11 +182,15 @@ public WMTSService(
 
         this.sb = sb;
         this.tld = tld;
+
         this.gsb = gsb;
         this.stats = stats;
         this.urlMangler = urlMangler;
         this.controller = controller;
         extensions.addAll(GeoWebCacheExtensions.extensions(WMTSExtension.class));
+        if (getMainConfiguration() != null) {
+            tld.setServiceInformation(getMainConfiguration().getServiceInformation());
```

Comment:
Having the WMTSService constructor responsible for setting the ServiceInformation in the Tile layer dispatcher seems like it's needlessly tying objects together.